zeebe/exporters/elasticsearch-exporter/src/main/java/io/camunda/zeebe/exporter/ElasticsearchExporterConfiguration.java

```diff
@@ -224,6 +226,7 @@ public static class IndexConfiguration {
     // index settings
     private Integer numberOfShards = null;
     private Integer numberOfReplicas = null;
+    private int templatePriority = DEFAULT_INDEX_TEMPLATE_PRIORITY;
```

Comment:
❓ what about adding this property to `broker.yaml.template` and `broker.standalone.yaml.template`?